### PR TITLE
Add description of new relationship operators

### PR DESCRIPTION
**Pull request by pontusmelke**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala [hunk 1]
```diff
@@ -643,6 +671,28 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     )
   }
 
+  @Test def directedRelationshipIndexSeek() {
+    profileQuery(
+      title = "Directed Relationship Index Seek",
+      text =
+        """The `DirectedRelationshipIndexSeek` operator finds relationships and their start and end nodes using an index seek.
+          |The relationship variable and the index used is shown in the arguments of the operator.""".stripMargin,
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:706]:
```suggestion
          |The relationship variable and the index used are shown in the arguments of the operator.""".stripMargin,
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala [hunk 2]
```diff
@@ -643,6 +671,28 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
...
+      assertions = p => assertThat(p.executionPlanDescription().toString, containsString("DirectedRelationshipIndexSeek"))
+    )
+  }
+
+  @Test def undirectedRelationshipIndexSeek() {
+    profileQuery(
+      title = "Relationship Index Seek",
+      text =
+        """The `UndirectedRelationshipIndexSeek` operator finds relationships and their start and end nodes using an index seek.
+          |The relationship variable and the index used is shown in the arguments of the operator.""".stripMargin,
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:717]:
```suggestion
          |The relationship variable and the index used are shown in the arguments of the operator.""".stripMargin,
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala [hunk 3]
```diff
@@ -742,6 +812,76 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     )
   }
 
+  @Test def directedRelationshipIndexScan() {
+    profileQuery(title = "Directed Relationship Index Scan",
+      text = """
+               |The `DirectedRelationshipIndexScan` operator examines all values stored in an index, returning all relationships and their start and end nodes with a particular relationship type having a specified property.""".stripMargin,
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:845]:
```suggestion
               |The `DirectedRelationshipIndexScan` operator examines all values stored in an index, returning all relationships and their start and end nodes with a particular relationship type and a specified property.""".stripMargin,
```

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala [hunk 4]
```diff
@@ -742,6 +812,76 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
...
+      queryText = "MATCH ()-[r: WORKS_IN]->() WHERE r.title IS NOT NULL RETURN r",
+      assertions = p => assertThat(p.executionPlanDescription().toString, containsString("DirectedRelationshipIndexScan"))
+    )
+  }
+
+  @Test def undirectedRelationshipIndexScan() {
+    profileQuery(title = "Undirected Relationship Index Scan",
+      text = """
+               |The `UndirectedRelationshipIndexScan` operator examines all values stored in an index, returning all relationships and their start and end nodes with a particular relationship type having a specified property.""".stripMargin,
```
Comment (AlexicaWright) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala:854]:
```suggestion
               |The `UndirectedRelationshipIndexScan` operator examines all values stored in an index, returning all relationships and their start and end nodes with a particular relationship type and a specified property.""".stripMargin,
```

**Review by AlexicaWright — COMMENTED** [review]
A few editorial comments, but mainly, this does not render properly. I can build it, but the section does not render.

**Comment by pontusmelke** [comment]
@AlexicaWright do you get any error message or such?

**Pull request merged**